extra/patterns/diamond.js

```diff
@@ -0,0 +1,27 @@
+var x=Number(process.argv[2]);
```

Comment:
You are using var here replace it with const.

Here are some guidelines that your code must be followed:-

- Spacing should be proper like
   let a = 123;
  if (a < 6 && a >3) {}
  for (i = 0; i < 5; i ++)

- Put the function calling after definition.
- Always start a new statement in new line, like
**wrong one**
if (a == 5) {a = a+1;
return a;
}
**correct one**
if (a == 5) {
a = a+1;
return a;
}